lib/galaxy/model/__init__.py

```diff
@@ -1135,6 +1135,8 @@ def to_dict(self, view='collection', system_details=False):
                     }
             rval['outputs'] = output_dict
 
+            rval['remappable'] = self.remappable()
```

Comment:
Not related to the build_for_rerun stuff but this would be handy for the rerun bioblend method

Reply:
This is likely an expensive method that we shouldn't call if it's not needed.
If necessary you could add this either in another view or as a separate endpoint maybe ?